### PR TITLE
Initial CI for analyzer V2

### PR DESCRIPTION
**Pull request by omri374**
Initial CI pipeline for V2 analyzer. Failing because it runs on the old analyzer

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+jobs:
+- job: 'Test'
+  pool:
+    vmImage: 'ubuntu-16.04'
+  strategy:
+    matrix:
+      Python37:
+        python.version: '3.7'
+#      Python38:
```
Comment (navalev) [inline azure-pipelines.yml:17]: should be also run on version 3.6?

**Review by navalev — DISMISSED** [review]
looks good for the initial pipeline, as discussed we will extend it in the next iteration. Let's commit to be used for additional PRs

File: azure-pipelines.yml (hunk 1 above)
Comment (omri374) [reply azure-pipelines.yml:17]: Possibly. Let's see how long it takes first and decide if we want to add more options?

Comment (navalev) [reply azure-pipelines.yml:17]: agree

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+        pipenv --python 3
+  - task: Bash@3
+    displayName: 'Install deps: Analyzer'
+    inputs:
+      targetType: 'inline'
+      workingDirectory: 'presidio-analyzer'
+      script: |
+        # Azure pipelines test publish
+        pipenv sync --dev --sequential
+        pipenv install --dev --skip-lock pytest-azurepipelines
```
Comment (balteravishay) [inline azure-pipelines.yml:43]: can this be taken to a different task so we can know which tasks are "core presidio" and which are "pipeline utility"?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+  - task: Bash@3
+    displayName: 'Pytest: Analyzer'
+    inputs:
+      targetType: 'inline'
+      workingDirectory: 'presidio-analyzer'
+      script: |
+        pipenv run pytest
+
+  - task: Bash@3
+    displayName: 'Linting: Analyzer'
```
Comment (balteravishay) [inline azure-pipelines.yml:55]: shouldn't lint be before test?

File: azure-pipelines.yml (hunk 2 above)
Comment (omri374) [reply azure-pipelines.yml]: Fixed. moved `pipenv install --dev --skip-lock pytest-azurepipelines` to the testing task

File: azure-pipelines.yml (hunk 3 above)
Comment (omri374) [reply azure-pipelines.yml]: Fixed

File: azure-pipelines.yml (hunk 2 above)
Comment (navalev) [reply azure-pipelines.yml]: wouldnt we replace the "core presidio" with a build script?

Comment (omri374) [reply azure-pipelines.yml]: I assume this is subject to change once we have a solid build script.

**Review by navalev — APPROVED** [review]

**Review by SharonHart — APPROVED** [review]

**Pull request merged**